pkgs/development/libraries/rapidcheck/default.nix

```diff
@@ -11,10 +11,12 @@ stdenv.mkDerivation rec {
     sha256 = "0n8l0mlq9xqmpkgcj5xicicd1my2cfwxg25zdy8347dqkl1ppgbs";
   };
 
-  nativeBuildInputs = [ cmake ];
+  nativeBuildInputs = [ cmake rsync ];
 
+  # Install the extras headers
   postInstall = ''
-    cp ../extras/boost_test/include/rapidcheck/boost_test.h $out/include/rapidcheck
+    cd $src
+    rsync --filter="- **CMakeLists.txt" -acRv extras $out
```

Comment:
Can you please use mv or cp?

Reply:
I originally had tried using `cp`, but you can't filter the files being copied.  Considering that it already depends on `cmake` I was thinking that a dependency on `rsync` at build time would be preferable to having installed the extras includes with the `CMakeLists.txt` files still lying around in there.  The thing I had tried was a recursive `cp` followed by an `rm **/CMakeLists.txt`, but the destination files were read only, so I switched.  If that approach would work by changing the file permissions in `$out` before doing the `rm`, then I'm happy to do that.

I played around with `builtins.filterSource` as well, but it isn't allowed to refer to store paths, so I was left wondering if there was a way to use a nix function to filter that directory, but wasn't sure what to try.

Reply:
I tried with the `chmod +w` and `rm` for the `CMakeLists.txt` files and it is producing the same result as the `rsync`, just in a more roundabout way.

Reply:
It is preferable to only use rsync if there is no other way. 4 Lines bash instead is fine for me.